subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -288,13 +288,65 @@ You can also assign collections or arrays of dependency notations to a configura
 [[sec:inspecting_dependencies]]
 === Inspecting dependencies
 
-You can generate dependency reports from the command line (see <<sec:listing_dependencies>>). With the help of the Project report plugin (see <<project_report_plugin>>) such a report can be created by your build.
+Gradle provides sufficient tooling to navigate large dependency graphs and mitigate situations that can lead to link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell]. Users can chose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency. Gradle offers both capabilities as command line tooling and through visual representation via build scans.
 
-Since Gradle 1.2 there is also a new programmatic API to access the resolved dependency information. The dependency reports (see the previous paragraph) are using this API under the covers. The API lets you walk the resolved dependency graph and provides information about the dependencies. In future releases the API will grow to provide more information about the resolution result. For more information about the API please refer to the Javadocs on api:org.gradle.api.artifacts.ResolvableDependencies#getResolutionResult[]. Potential usages of the api:org.gradle.api.artifacts.result.ResolutionResult[] API:
+[[sec:listing_dependencies]]
+==== Listing all dependencies in a project
 
-* Creation of advanced dependency reports tailored to your use case.
-* Enabling the build logic to make decisions based on the content of the dependency graph.
+A project can declare one or more dependencies. Gradle can visualize the whole dependency tree for every configuration available in the project with the help of the _dependency report_.
 
+Rendering the dependency tree is particularly useful if you'd like identify which dependencies have been resolved at runtime. It also provides you with information about any dependency conflict resolution that occurred in the process and clearly indicates the selected version. The dependency report always contains declared and transitive dependencies.
+
+Let's say you'd want to create tasks for your project that use the link:https://www.eclipse.org/jgit/[JGit library] to execute SCM operations e.g. to model a release process. You can declare dependencies for any external tooling with the help of a <<sub:scope_of_dependency_configurations,custom configuration>> so that it doesn't doesn't convolute other contexts like the compilation classpath for your production source code. To render the dependencies graph execute the `dependencies` task for the corresponding configuration.
```

Comment:
⭕️  I think "pollute" is better than "convolute" in this context.

Reply:
done